lib/Wikia/src/Service/Swagger/ApiClient.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Wikia\Service\Swagger;
+
+use Swagger\Client\ApiException;
+use Swagger\Client\Configuration;
+use Wikia\Logger\Loggable;
+use Wikia\Util\Statistics\BernoulliTrial;
+
+class ApiClient extends \Swagger\Client\ApiClient {
+
+	use Loggable;
+
+	/** @var BernoulliTrial */
+	private $logSampler;
+
+	/** @var string */
+	private $serviceName;
+
+	public function __construct(Configuration $config, BernoulliTrial $logSampler, $serviceName) {
+		parent::__construct($config);
+		$this->logSampler = $logSampler;
+		$this->serviceName = $serviceName;
+	}
+
+	public function callApi($resourcePath, $method, $queryParams, $postData, $headerParams, $responseType=null) {
+		$start = microtime(true);
+		$response = $exception = null;
+		$code = 200;
+
+		try {
+			$response = parent::callApi($resourcePath, $method, $queryParams, $postData, $headerParams, $responseType);
```

Comment:
Oh, I see. It's probably fine so long as we don't need to search by `statusCode`. 

We might want to take it out since we also have `isOk` which conveys similar information.